src/extensions/default/Inline3DParametersEditor/Parameters3DUtils.js

```diff
@@ -0,0 +1,35 @@
+/**
+ *  Utilities functions related to 3-D parameter matching
+ *
+ */
+define(function (require, exports, module) {
+    "use strict";
+
+    var PARAMETERS_3D_REGEX = /( )*(((-)?\d+(\.\d+)?)( )+)*((-)?\d+(\.\d+)?( )*)/g;
+    var TAG_START = "a-";
+
+    var MODIFIERS = {
+        position: 50,
+        scale: 30,
+        rotation: 5,
+        radius: 50
+    };
+
+    /*
+     * Returns the scroll modifier corresponding to a parameter.
+     * @param {?string} tag : tag corresponding to which modifier
+     * is required
+     * @return Modifier value
+     */
+    function getModifier(tag) {
+        if(MODIFIERS.hasOwnProperty(tag)) {
+            return MODIFIERS[tag];
+        }
+        return 25;
```

Comment:
what is 25? Needs to be assigned to a variable or some documentation on what it means.